101-aks-advanced-networking-aad/azuredeploy.parameters.json

```diff
@@ -0,0 +1,42 @@
+    {
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "dnsPrefix": {
+            "value": "GEN-UNIQUE"
+        },
+        "existingServicePrincipalObjectId": {
+            "value": "GEN-GUID"
+        },
+        "existingServicePrincipalClientId": {
+            "value": "GEN-GUID"
+        },
+        "existingServicePrincipalClientSecret": {
+            "value": "GEN-PASSWORD"
+        },
+        "existingVirtualNetworkName": {
+            "value": "GET-PREREQ-vnetName"
+        },
+        "existingVirtualNetworkResourceGroup": {
+            "value": "GET-PREREQ-vnetResourceGroupName"
+        },
+        "existingSubnetName": {
+            "value": "GET-PREREQ-subnetName"
+        },
+        "AAD_ClientAppID":{
+            "value": "GET-CLIENT-APP-ID-FROM-AZURE-AD"
```

Comment:
the GETs here are not valid for CI/CD so just remove them - since we're skipping CI it won't matter